### DISCUSSION TITLE
Everyone join in the harso-css organization

### DISCUSSION THREAD

**Discussion by Sreehari521** [Sreehari521/harso-css#9] [Announcements]
@Adatta1276 @Genius398 @Super45coder @somePythonProgrammer Everyone join in the harso-css organization. Contribute it there

**Comment by devashish2024** [comment]
btw, invite us!!

**Comment by dolphin007x** [comment]
@Sreehari521 I won't be able to join as I have a project of my own that I'm working on

**Reply by dolphin007x** [reply]
Like I'm using APIs for website customisation and it's really tough 🤔😐

**Comment by devashish2024** [comment]
YAY!!!

**Comment by Super45coder** [comment]
I can join, but i am not that skilled at creating libraries.
________________________________

**Comment by Genius398** [comment]
STOP PINGING MEEEEE

**Reply by devashish2024** [reply]
uhh, sorry for pinging you...

**Comment by Super45coder** [comment]
Sorry
________________________________
